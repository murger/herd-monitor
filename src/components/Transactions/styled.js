import styled from 'styled-components';

export const List = styled.ul`
  width: 100%;
  margin: 0px;
  padding: 32px;
  box-sizing: border-box;
  list-style: none;
`;

export const Item = styled.li`
	padding: 8px 0;
`;
