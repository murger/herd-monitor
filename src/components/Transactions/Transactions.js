import React from 'react';
import { List, Item } from './styled';

const Transactions = () => {
	return (
		<List>
			<Item>Alpha</Item>
			<Item>Beta</Item>
			<Item>Gamma</Item>
		</List>
	);
};

export default Transactions;
