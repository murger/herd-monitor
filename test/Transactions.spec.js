import React from 'react';
import renderer from 'react-test-renderer';
import Transactions from '../src/components/Transactions';

it('renders the correctly', () => {
  const tree = renderer.create(<Transactions />).toJSON();
  expect(tree).toMatchSnapshot();
});
