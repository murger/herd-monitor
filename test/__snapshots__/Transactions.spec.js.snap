// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the correctly 1`] = `
<ul
  className="sc-bdVaJa cBsPko"
>
  <li
    className="sc-bwzfXH iOBHAi"
  >
    Alpha
  </li>
  <li
    className="sc-bwzfXH iOBHAi"
  >
    Beta
  </li>
  <li
    className="sc-bwzfXH iOBHAi"
  >
    Gamma
  </li>
</ul>
`;
